indexing-hadoop/src/test/java/io/druid/indexer/BatchDeltaIngestionTest.java

```diff
@@ -70,8 +70,9 @@
   TemporaryFolder temporaryFolder = new TemporaryFolder();
 
   private ObjectMapper mapper;
-  private Interval interval;
-  private List<DataSegment> segments;
+  private Interval INTERVAL_FULL = new Interval("2014-10-22T00:00:00Z/P1D");
+  private Interval INTERVAL_PARTIAL = new Interval("2014-10-22T00:00:00Z/PT2H");
+  private DataSegment SEGMENT;
```

Comment:
since they are non-static, can we call them intervalFull, intervalPartial and segment ?